DSharpPlus/Entities/Interaction/Components/DiscordSelectComponent.cs

```diff
@@ -68,7 +62,7 @@ public DiscordSelectComponent()
             this.Type = ComponentType.Select;
         }
 
-        public DiscordSelectComponent(string customId, IEnumerable<DiscordSelectComponentOption> options, string placeholder = null, int minOptions = 1, int maxOptions = 1)
+        public DiscordSelectComponent(string customId, string placeholder = null, IEnumerable<DiscordSelectComponentOption> options = default, int minOptions = 1, int maxOptions = 1) : this()
```

Comment:
I don't think options are an optional parameter

Reply:
Should be fixed